test/ctia/auth/jwt_test.clj

```diff
@@ -1,20 +1,24 @@
 (ns ctia.auth.jwt-test
   (:require [ctia.auth.jwt :as sut]
             [ctia.auth.capabilities :as caps]
+            [ctia.properties :as p]
             [clojure.test :as t :refer [deftest is]]
             [clojure.set :as set])
   (:import [ctia.auth.jwt JWTIdentity]))
 
+(def get-in-config
+  p/get-in-global-properties)
```

Comment:
Above is a placeholder for creating a dummy `get-in-config` function. Since there's no fixtures in this namespace, there'll be no need to boot up a TK app, so we have to construct our own `get-in-config`.